actionpack/test/dispatch/session/mem_cache_store_test.rb

```diff
@@ -40,7 +40,7 @@ def call_reset_session
     require "dalli"
     servers = ENV["MEMCACHE_SERVERS"] || "localhost:11211"
     ss = Dalli::Client.new(servers).stats
-    raise Dalli::DalliError unless ss[servers]
+    raise Dalli::DalliError unless ss[servers] || ss[servers + ":11211"]
```

Comment:
I originally was going to do something where I checked if `servers` already contained a port and conditionally appended it, but this seemed much simpler.